build.sbt

```diff
@@ -137,32 +147,24 @@ lazy val assemblySettings = Seq(
     case s if s.endsWith("snappyjava_snappy.dll") => MergeStrategy.last
     case s if s.endsWith(".dtd")                  => MergeStrategy.rename
     case s if s.endsWith(".xsd")                  => MergeStrategy.rename
+    case PathList("META-INF", "NOTICE")           =>
+      // avro-tools META-INF/NOTICE must not be renamed
+      CustomMergeStrategy.rename {
+        case l: Assembly.Library if l.moduleCoord.name == "avro-tools" =>
+          l.target
+        case l: Assembly.Library =>
+          l.target + "_" + l.moduleCoord.name + "-" + l.moduleCoord.version
+        case p: Assembly.Project =>
+          p.target + "_" + p.name
+      }
     case PathList("META-INF", "services", "org.apache.hadoop.fs.FileSystem") =>
       MergeStrategy.filterDistinctLines
-    case PathList("META-INF", "LICENSE")     => MergeStrategy.discard
-    case PathList("META-INF", "MANIFEST.MF") => MergeStrategy.discard
-    case PathList("META-INF", "INDEX.LIST")  => MergeStrategy.discard
+    case PathList("META-INF", "LICENSE")               => MergeStrategy.discard
+    case PathList("META-INF", "MANIFEST.MF")           => MergeStrategy.discard
+    case PathList("META-INF", "INDEX.LIST")            => MergeStrategy.discard
     case PathList("META-INF", s) if s.endsWith(".DSA") => MergeStrategy.discard
     case PathList("META-INF", s) if s.endsWith(".RSA") => MergeStrategy.discard
     case PathList("META-INF", s) if s.endsWith(".SF")  => MergeStrategy.discard
-    case PathList("META-INF", "NOTICE")      => MergeStrategy.rename
```

Comment:
I have no idea how this worked before. `avro-tool` is reading the `NOTICE` file on [startup](https://github.com/apache/avro/blob/9738a759f01e267772092c2299d8f0d7047942a8/lang/java/tools/src/main/java/org/apache/avro/tool/Main.java#L75). Renaming should not have worked.

Reply:
I get it. it was replaced by our own one